Tweets/_posts/What is Yearn/2021-10-09-What-is-Yearn.md

```diff
@@ -4,95 +4,112 @@ title:  "What is Yearn"
+
+## 11/n
+
+A pesar de todos los protocolos interesantes hasta ahora, aquellos que traen la mayor adopción a Yearn probablemente aún no se hayan desarrollado, pero hay muchos equipos trabajando en ellos. [pic.twitter.com/cIHKjDL7Kr](https://twitter.com/Wot_Is_Goin_On/status/1446540036019736576/photo/1)
 
 ![3_1446535242404274176](3_1446535242404274176.jpg)
 
-##### 12
-With Yearn going multi-chain there’s potential for different types of dapps on each chain. [twitter.com/iearnfinance/s…](https://twitter.com/iearnfinance/status/1446243257336229912?s=20)
+## 12/n
+
+Ahora que Yearn será multicadena, existe la posibilidad de diferentes tipos de dapps en cada cadena. [twitter.com/iearnfinance/s…](https://twitter.com/iearnfinance/status/1446243257336229912?s=20)
 
 > ![iearnfinance](earnfinance-1223779978459770880.jpg)
 > yearn.finance ([@iearnfinance](https://twitter.com/iearnfinance))
 
-> The Realm of Multichain beckons.
+> El Reino Multicadena llama.
 
-And Yearn has heeded the call. 📯
+Y Yearn ha atendido el llamado. 📯
 
-Today, we go multichain with the launch of Iron Bank Fantom and the first Fantom vaults on [beta.yearn.finance](http://beta.yearn.finance)🧵
+Hoy, nos volvemos multicadena con el lanzamiento de Iron Bank Fantom y las primeras bóvedas de Fantom en [beta.yearn.finance](http://beta.yearn.finance)🧵 [pic.twitter.com/IaAtDqdkYi](https://twitter.com/iearnfinance/status/1446243257336229912/photo/1)
 
 > ![3_1446243220468293632](3_1446243220468293632.jpg)
 
-##### 13
-The people calling for the death of DeFi on Ethereum may need to zoom out. Yearn’s Q3 2021 TVL was $4.6bn, nearly 300% higher than 12 months before. Q3 2021 annualised revenue was over $60m, up nearly 400% over the year.
+## 13/n
+
+Aquellos que creen que Defi en Ethereum está muerto deberían reconsiderarlo. El TVL del tercer trimestre de 2021 de Yearn $4.6MM casi un 300% más que 12 meses antes. Los ingresos anualizados del tercer trimestre de 2021 superaron los $60M, casi un 400% más anual. [pic.twitter.com/T2ivxDR0Ek](https://twitter.com/Wot_Is_Goin_On/status/1446540042210512906/photo/1)
 
 ![3_1446536804035960832](3_1446536804035960832.jpg)
 
-##### 14
-The criticism around lack of value accrual is one that can be easily fixed. However, the challenge is to design tokenomics that incentivize growth and sustainability of the protocol in terms of contributors and TVL, as well as value accrual.
+## 14/n
 
-##### 15
-It took Ethereum 5+ years to implement EIP-1559, and 2022 brings the migration to PoS that will reward ETH stakers instead of miners. And when cartoon animals are incentivized to figure things out, things usually get figured out.
+La crítica en torno a la falta de acumulación de valor se puede solucionar fácilmente. Sin embargo, el desafío es diseñar una economía de token que incentive el crecimiento y la sostenibilidad del protocolo en términos de contribuyentes y TVL, así como la acumulación de valor.
```

Comment:
AS-IS: "y la sostenibilidad del protocolo en términos de contribuyentes y TVL, así como la acumulación de valor."

TO-BE: "y la sostenibilidad del protocolo en términos de colaboradores y TVL, así como la acumulación de valor."

I've replaced the word "contribuyentes" by "collaboradores" due to that "contribuyentes" in Spanish can have a meaning of "tax payers" which is not the case here, what we want to communicate is more collaborators = contributors. The collaborators word is Spanish provides a better meaning in this context.

Reply:
fixed